roles/debian/nginx/defaults/main.yml

```diff
@@ -7,6 +7,8 @@ php:
 symfony_env: "{{ _env_type }}"
 # Nginx variables actually start here.
 nginx:
+  http_port: 80
```

Comment:
We don't need this at all, it's already handled at vhost level. :-)

Reply:
resolved:)